<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search Injury</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TITLE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1021</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter the type of Injury</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>1001</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
  background-color: white;
color: black;
border-style: outset;
border-width: 2px;
border-radius:20px;
border-color: black;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_button">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>190</y>
      <width>280</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>140</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed
{
  background-color: rgb(195,30,60);
}

QPushButton
{
  background-color: white;
color: black;
border-style: outset;
border-width: 3px;
border-radius:10px;
border-color: black;
}</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>340</y>
      <width>1071</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>1024x222 keyboard pi.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_injury_go_back_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>280</width>
      <height>140</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>140</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover:!pressed
{
  background-color: rgb(195,30,60);
}

QPushButton
{
  background-color: white;
color: black;
border-style: outset;
border-width: 3px;
border-radius:10px;
border-color: black;
}</string>
    </property>
    <property name="text">
     <string>Go back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>600</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pictures/program_pictures/main_wallpaper.png</pixmap>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>TITLE</zorder>
   <zorder>lineEdit</zorder>
   <zorder>enter_button</zorder>
   <zorder>label</zorder>
   <zorder>enter_injury_go_back_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
